airflow/api_connexion/openapi/v1.yaml

```diff
@@ -2442,6 +2442,7 @@ components:
           enum:
             - success
             - failed
+            - queued
```

Comment:
I guess this probably needs a note in description that this value was added in 2.3.1 or 2.4.0?

Reply:
This is a good point. I think we lack a bit of coordination on the API versioning policies. Created an issue for that. https://github.com/apache/airflow/issues/23585